gtk/CMakeLists.txt

```diff
@@ -188,6 +198,7 @@ endif()
 
 if(ENABLE_NLS)
     install(FILES ${${PROJECT_NAME}_DESKTOP_FILE} DESTINATION ${CMAKE_INSTALL_DATADIR}/applications)
+    install(FILES ${${PROJECT_NAME}_APPDATA_FILE} DESTINATION ${CMAKE_INSTALL_DATADIR}/appdata)
```

Comment:
The documentation link you provided mentions /usr/share/metainfo/ and not /usr/share/appdata/, so is there a location where each and every distribution will find this file?